### PR TITLE
Adding an automated test for this control. 

### PR DESCRIPTION
**Pull request by djhaynes**
This PR is associated with PR #8.

### CODE REVIEW

File: controls/V-72315.rb [hunk 1]
```diff
@@ -77,7 +147,48 @@
...
+    describe package('tcp_wrappers') do
+      it { should be_installed }
+    end
+    tcpwrappers_allow.each do |rule|
+      describe etc_hosts_allow.where { daemon == rule['daemon'] } do
+        its('client_list') { should be rule['client_list'] }
+        its('options') { should be rule['options'] }
+      end
+    end
+    tcpwrappers_deny.each do |rule|
```
Comment (trevor-vaughan) [inline controls/V-72315.rb]: `/etc/hosts.deny` shouldn't exist at all should it?

Comment (djhaynes) [reply controls/V-72315.rb]: I think we need that check. The control's check-text says:

"If \"firewalld\" is not active, determine whether \"tcpwrappers\" is being used
by checking whether the \"hosts.allow\" and \"hosts.deny\" files are empty with
the following commands:"

Comment (trevor-vaughan) [reply controls/V-72315.rb]: @djhaynes Ah, right...the default for `/etc/hosts.deny` should be `ALL: ALL`. Had to re-read the man page.

Comment (djhaynes) [reply controls/V-72315.rb]: Given this, I am going to merge in this control.

**Pull request merged**